### ISSUE TITLE
[CI] ClusterBlockException causes docs update-by-query test to fail

### ISSUE THREAD

**Issue by dimitris-athanasiou** [elastic/elasticsearch#27673]
Link to build:
```
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.1+oracle-periodic/155/consoleFull
```

Reproduce:
```
gradle :docs:integTestRunner -Dtests.seed=6EF7C444DB683D8E -Dtests.class=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT -Dtests.method="test {yaml=reference/docs/update-by-query/line_625}" -Dtests.security.manager=true -Dtests.locale=ar-SY -Dtests.timezone=Australia/Broken_Hill
```

Failure:
```
> Throwable #1: java.lang.AssertionError: Failure at [reference/docs/update-by-query:1414]: expected [2xx] status code but api [raw[method=POST path=test/test]] returned [403 Forbidden] [{"error":{"root_cause":[{"type":"cluster_block_exception","reason":"blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];","stack_trace":"ClusterBlockException[blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];]\n\tat org.elasticsearch.cluster.block.ClusterBlocks.indexBlockedException(ClusterBlocks.java:182)\n\tat org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase.handleBlockExceptions(TransportReplicationAction.java:812)\n\tat org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase.doRun(TransportReplicationAction.java:712)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\n\tat org.elasticsearch.action.support.replication.TransportReplicationAction.doExecute(TransportReplicationAction.java:169)\n\tat org.elasticsearch.action.support.replication.TransportReplicationAction.doExecute(TransportReplicationAction.java:97)\n\tat org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:167)\n\tat org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:139)\n\tat org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:81)\n\tat org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation.doRun(TransportBulkAction.java:350)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\n\tat org.elasticsearch.action.bulk.TransportBulkAction.executeBulk(TransportBulkAction.java:462)\n\tat org.elasticsearch.action.bulk.TransportBulkAction.doExecute(TransportBulkAction.java:175)\n\tat org.elasticsearch.action.bulk.TransportBulkAction.doExecute(TransportBulkAction.java:86)\n\tat org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:167)\n\tat org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:139)\n\tat org.elasticsearch.action.bulk.TransportSingleItemBulkWriteAction.doExecute(TransportSingleItemBulkWriteAction.java:69)\n\tat org.elasticsearch.action.bulk.TransportSingleItemBulkWriteAction.doExecute(TransportSingleItemBulkWriteAction.java:44)\n\tat org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:167)\n\tat org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:139)\n\tat org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:81)\n\tat org.elasticsearch.client.node.NodeClient.executeLocally(NodeClient.java:83)\n\tat org.elasticsearch.client.node.NodeClient.doExecute(NodeClient.java:72)\n\tat org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:405)\n\tat org.elasticsearch.client.support.AbstractClient.index(AbstractClient.java:417)\n\tat org.elasticsearch.rest.action.document.RestIndexAction.lambda$prepareRequest$1(RestIndexAction.java:92)\n\tat org.elasticsearch.rest.BaseRestHandler.handleRequest(BaseRestHandler.java:97)\n\tat org.elasticsearch.rest.RestController.dispatchRequest(RestController.java:240)\n\tat org.elasticsearch.rest.RestController.tryAllHandlers(RestController.java:336)\n\tat org.elasticsearch.rest.RestController.dispatchRequest(RestController.java:174)\n\tat org.elasticsearch.http.netty4.Netty4HttpServerTransport.dispatchRequest(Netty4HttpServerTransport.java:497)\n\tat org.elasticsearch.http.netty4.Netty4HttpRequestHandler.channelRead0(Netty4HttpRequestHandler.java:80)\n\tat io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)\n\tat 
```

Could not reproduce locally. Seems tricky as it's probably some other test that caused the block.

**Comment by nik9000** [comment]
This is a duplicate of #28325. Probably. I'm going to spend some more time hunting this down with more data points.
